src/app/app.module.ts

```diff
@@ -59,9 +60,32 @@ import { ComputeResourcePropertyTypesListComponent } from './components/compute-
 import { AddOrEditComputeResourcePropertyTypeDialogComponent } from './components/compute-resource-property-types/dialogs/add-or-edit-compute-resource-property-type-dialog/add-or-edit-compute-resource-property-type-dialog.component';
 import { QcAtlasUiFeatureToggleModule } from './directives/feature-toggle.module';
 import { FeatureTogglingComponent } from './components/feature-toggling/feature-toggling.component';
-import { PlanqkPlatformLoginDialogComponent } from './components/dialogs/planqk-platform-login-dialog.component';
 import { AuthInterceptor } from './http-interceptors/auth-interceptor';
 
+const initializeKeycloak = (keycloak: KeycloakService) => (): Promise<
```

Comment:
Please move this function into an extra file such that it is not contained in the app.module

Reply:
see, e.g., https://blog.bitsrc.io/integrating-keycloak-with-angular-for-sso-authentication-9d1c6c2d2742

Reply:
done